src/app/lists/ListDataview.tsx

```diff
@@ -323,6 +316,12 @@ const ListsDataView: React.FC<ListDataViewProps> = ({
                         additionalFiltersShown: false,
                     }}
                 />
+                <EditModal
+                    onClose={() => setModal(undefined)}
+                    data={modal}
+                    key={modal?.primary_key}
+                    currentList={primaryFilters[1].state === "hotel" ? "hotel" : "regular"}
```

Comment:
This looks like a hold-over from capitalisation problems, can this just be `primaryFilters[1].state`?